api/answers/views.py

```diff
@@ -129,3 +129,43 @@ def put(self, request, meetupId, questionId, answerId):
         except ValidationError:
             error = {'error': 'The specified answer does not exist'}
             return Response(data=error, status=status.HTTP_404_NOT_FOUND)
+
+
+class DeleteAnswer(APIView):
+    """
+    Class to delete a specific answer
+    """
+    permission_classes = [permissions.IsAuthenticated]
+
+    def delete(self, request, meetupId, questionId, answerId):
+        """
+        Endpoint to deleting specific answer
+        Delete /api/meetups/{meetupId}/questions/{questionId}/answers/{answerId}/
+        """
+        try:
+            meetid = Meetup.objects.get(id=meetupId)
+        except  (ObjectDoesNotExist, ValidationError) as e:
+            error = {'error': str(e)}
+            return Response(data=error, status=status.HTTP_404_NOT_FOUND)
+
+        try:
+            questionid = Question.objects.get(id=questionId)
+        except  (ObjectDoesNotExist, ValidationError) as e:
+            error = {'error': str(e)}
+            return Response(data=error, status=status.HTTP_404_NOT_FOUND)
+
+        try:
+            answer = Answer.objects.get(id=answerId)
+            userid = request.user.id
+            if userid != answer.creator.id:
```

Comment:
Ensure to also validate that if a user is an admin,  the user can delete any answer posted.

Reply:
@IssaIan  i fixed it